journal.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+from pyramid.config import Configurator
+from pyramid.view import view_config
+from waitress import serve
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+import datetime
+
+
+Base = declarative_base()
+
+
+class Entry(Base):
+
+    __tablename__ = "entries"
+
+    id = sa.Column(sa.Integer, primary_key=True, autoincrement=True)
+    title = sa.Column(sa.Unicode(255), nullable=False)
+    creation_date = sa.Column(sa.DateTime, nullable=False, default=datetime.datetime.utcnow)
```

Comment:
This line should be causing a linter warning in your editor.  It's certainly longer than 80 chars.  The same is true for line 30 below.  Please fix.